sageodata-apispec.yml

```diff
@@ -2351,21 +2352,82 @@ components:
       properties:
         unitNumber:
           format: int32
+          description: A combination of the map100000Number and sequence number.
           type: integer
         nrmRegionCode:
           description: NRM region code
+          enum:
+          - Eyre
+          - Peninsula
+          - South
+          - Australian
+          - Arid
+          - Lands
+          - East
+          - Northern
+          - '&'
+          - Yorke
+          - Alinytjara
+          - Wilurara
+          - Adelaide
+          - Mt
+          - Lofty
+          - Ranges
+          - Murray-Darling
+          - Basin
+          - Kangaroo
+          - Island
           type: string
         prescribedWellArea:
+          description: Prescribed well area code.
+          enum:
+          - Musgrave
+          - Southern
+          - Basins
+          - McLaren
+          - Vale
+          - Northern
+          - Adelaide
+          - Plains
+          - Noora
+          - Padthaway
+          - Central
+          - Far
+          - North
+          - Mallee
+          - Peake
+          - Roby
+          - and
+          - Sherlock
+          - Tintinara-Coonalpyn
+          - Lower
+          - Limestone
+          - Coast
+          - Angas-Bremer
+          - Tatiara
```

Comment:
Each word in here has now been defined as a valid enum value. For eg: 

> and, 
> Coast etc.

I would recommend using a key as the enum value and provide a description as to what it signifies.
An example would be the _testMethod_ variable in the _wcr-pumptestdata_

Reply:
SAGeodata currently has the following values in _DD_PRESCRIBED_WELL_AREA_ table
|Code|Value|
|---|---|
|Angas-Bremer	|	Angas-Bremer Prescribed Wells Area|
|Central Adelaide	|	Central Adelaide Groundwater Area|
|Far North	|	Far North Prescribed Wells Area|
|Lower Limestone Coast	|	Lower Limestone Coast Prescribed Wells Area|
|Mallee	|	Mallee Prescribed Wells Area|
|McLaren Vale	|	McLaren Vale Prescribed Wells Area|
|Musgrave	|	County Musgrave Prescribed Wells Area|
|Noora	|	Noora Prescribed Wells Area|
|Northern Adelaide Plains	|	Northern Adelaide Plains Prescribed Wells Area|
|Padthaway	|	Padthaway Prescribed Wells Area|
|Peake, Roby and Sherlock	|	Peake, Roby and Sherlock Prescribed Wells Area|
|Southern Basins	|	Southern Basins Prescribed Wells Area|
|Tatiara	|	Tatiara Prescribed Wells Area|
|Tintinara-Coonalpyn	|	Tintinara-Coonalpyn Prescribed Wells Area|
|Dry Creek	|	Dry Creek Prescribed Wells Area|